--- ./setup/manageheader.php	2005-06-16 08:36:09.000000000 +0300
+++ /tmp/manageheader.php	2005-09-11 15:47:35.000000000 +0300
@@ -38,7 +38,7 @@
 		}
 		else
 		{
-			$realpath = '/path/to/egroupware';
+			$realpath = '/usr/share/egroupware';
 		}
 	}
 
@@ -360,7 +360,7 @@
 					$setup_tpl->set_var('db_name','egroupware');
 					$setup_tpl->set_var('db_user','egroupware');
 					$setup_tpl->set_var('db_pass','');
-					$setup_tpl->set_var('config_user','changeme');
+					$setup_tpl->set_var('config_user','config');
 					$setup_tpl->set_var('config_pass','');
 					while(list($k,$v) = @each($supported_db))
 					{
@@ -471,7 +471,7 @@
 				$setup_tpl->set_var('db_name','egroupware');
 				$setup_tpl->set_var('db_user','egroupware');
 				$setup_tpl->set_var('db_pass','');
-				$setup_tpl->set_var('config_user','changeme');
+				$setup_tpl->set_var('config_user','config');
 				$setup_tpl->set_var('config_pass','');
 
 				while(list($k,$v) = each($supported_db))
--- ./setup/index.php	2004-11-17 14:48:22.000000000 +0200
+++ /tmp/index.php	2005-09-11 16:07:11.000000000 +0300
@@ -177,7 +177,7 @@
 					$setup_tpl->set_var('instr',
 						lang("Instructions for creating the database in %1:",'MySql')
 						. '<br>'.lang('Login to mysql -')
-						. '<br><i>[user@server user]# mysql -u root -p</i><br>'
+						. '<br><i>[user@server user]# mysql -u mysql -p</i><br>'
 						. lang('Create the empty database and grant user permissions -')
 						. "<br><i>mysql> create database $info[db_name];</i>"
 						. "<br><i>mysql> grant all on " . $info['db_name']
