--- ./setup/manageheader.php	2005-06-16 08:36:09.000000000 +0300
+++ /tmp/manageheader.php	2005-09-11 15:47:35.000000000 +0300
@@ -38,7 +38,7 @@
 		}
 		else
 		{
-			$realpath = '/path/to/egroupware';
+			$realpath = '/usr/share/egroupware';
 		}
 	}
 
@@ -360,7 +360,7 @@
 					$setup_tpl->set_var('db_name','egroupware');
 					$setup_tpl->set_var('db_user','egroupware');
 					$setup_tpl->set_var('db_pass','');
-					$setup_tpl->set_var('config_user','changeme');
+					$setup_tpl->set_var('config_user','config');
 					$setup_tpl->set_var('config_pass','');
 					while(list($k,$v) = @each($supported_db))
 					{
@@ -471,7 +471,7 @@
 				$setup_tpl->set_var('db_name','egroupware');
 				$setup_tpl->set_var('db_user','egroupware');
 				$setup_tpl->set_var('db_pass','');
-				$setup_tpl->set_var('config_user','changeme');
+				$setup_tpl->set_var('config_user','config');
 				$setup_tpl->set_var('config_pass','');
 
 				while(list($k,$v) = each($supported_db))
--- ./setup/index.php	2004-11-17 14:48:22.000000000 +0200
+++ /home/builder/rpm/BUILD/egroupware/setup/index.php	2006-02-05 19:31:35.000000000 +0200
@@ -201,7 +201,7 @@
 					$setup_tpl->set_var('instr',
 						'<b>'.lang("Instructions for creating the database in %1:",'MySql').'</b>'
 						. '<br />'.lang('Login to mysql -')
-						. '<br /><i>[user@server user]# <b>mysql -u root -p</b></i><br />'
+						. '<br /><i>[user@server user]# <b>mysql -u mysql -p</b></i><br />'
 						. lang('Create the empty database and grant user permissions -')
 						. "<br /><i>mysql> <b>CREATE DATABASE $info[db_name]$set_charset;</b></i>"
 						. "<br /><i>mysql> <b>GRANT ALL ON " . $info['db_name']
